### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by sidph**
Replaced supported resource section with a table to show new capabilities

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/account_management/rbac/_index.md [hunk 1]
```diff
@@ -181,17 +181,20 @@ When creating or updating a role on the Datadog site, use a Datadog role templat
 
 {{< img src="account_management/rbac/role_templates.png" alt="Role Templates dropdown menu with Datadog Billing Admin Role selected" style="width:90%;">}}
 
-## Restrict access to individual resources
+## Restrict access to individual resources (Granular Access Control)
```
Comment (drichards-87) [inline content/en/account_management/rbac/_index.md:184]:
```suggestion
## Restrict access to individual resources (granular access controls)
```

File: content/en/account_management/rbac/_index.md [hunk 2]
```diff
@@ -181,17 +181,20 @@ When creating or updating a role on the Datadog site, use a Datadog role templat
...
+
+
+| Supported Resources with Granular Access Control  | Team based Access |  Role based Access    |  User / Service Account Based Access  |
+|-------------------------|-----------------------------------|---------------------|--------------------------|
+| [Dashboards][11]       | **✓**                 |      **✓**                     |        **✓**                    |
+| [Monitors][12]   |                                                     |       **✓**           |
+| [Notebooks][6]   | **✓**                   |            **✓**              |         **✓**                   |
+| [Security rules][13]         | **✓**      |        **✓**                    |       **✓**                                |
+| [Service Level Objectives][14]   |  **✓**    | **✓**                         |     **✓**                                        |
+| [Synthetic tests][15] | |**✓**  |   |
```
Comment (drichards-87) [inline content/en/account_management/rbac/_index.md:196]:
```suggestion
| Supported Resources with Granular Access Control | Team-Based Access | Role-Based Access | User / Service Account-Based Access |
|--------------------------------------------------|-------------------|-------------------|-------------------------------------|
| [Dashboards][11]                                 | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
| [Monitors][12]                                   |                   | {{< X >}}         |                                     |
| [Notebooks][6]                                   | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
| [Security rules][13]                             | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
| [Service Level Objectives][14]                   | {{< X >}}         | {{< X >}}         | {{< X >}}                           |
| [Synthetic tests][15]                            |                   | {{< X >}}         |                                     |
```

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left you some minor feedback.

**Review by drichards-87 — APPROVED** [review]
Approved. Is this ready to merge now?

**Comment by sidph** [comment]
> 

Yes, this looks good!

**Pull request merged**